imageroot/bin/DB-fix

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+#
+# Copyright (C) 2025 Nethesis S.r.l.
+# http://www.nethesis.it - nethserver@nethesis.it
+#
+# This script is part of NethServer.
+#
+# NethServer is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License,
+# or any later version.
+#
+# NethServer is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with NethServer.  If not, see COPYING.
+#
+
+# Fix row format
+if occ setupchecks | grep -q 'MySQL row format: Incorrect'; then
+  echo "Fix DB row format..."
+  # Create temp sql file
+  tmpsql=$(mktemp dbfix.XXXXXX)
+  trap 'rm -f ${tmpsql}' EXIT
+
+  # Write sql file
+  podman exec nextcloud-db mysql -u nextcloud -pnextcloud -Bse "SELECT CONCAT('ALTER TABLE \`', TABLE_NAME, '\`ROW_FORMAT=DYNAMIC;') FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_SCHEMA = 'nextcloud' AND ENGINE = 'InnoDB';" > ${tmpsql}
```

Comment:
can we add some comment for what it is needed, what version issue it is intended to fix (from which version)

Reply:
question, can we run it multi times ?

Reply:
I'm going to add comments that explain need and version.
Yes, it can run multiple times. It's checked via occ if the fix is needed and only applied when needed. So a second run does nothing when the patches are already applied.